File: client/ios/Contract/Attachment/AttachmentUploadHandoffRequest.swift
import Foundation

public struct AttachmentUploadHandoffRequest: Sendable {
    public let ownerType: String
    public let ownerId: String
    public let context: String?
    public let slot: String?
    public let isPrimary: Bool
    public let title: String?
    public let description: String?
    public let altText: String?

    public init(ownerType: String, ownerId: String, context: String?, slot: String?, isPrimary: Bool, title: String?, description: String?, altText: String?) {
        self.ownerType = ownerType
        self.ownerId = ownerId
        self.context = context
        self.slot = slot
        self.isPrimary = isPrimary
        self.title = title
        self.description = description
        self.altText = altText
    }
}

File: client/android/Contract/Attachment/AttachmentUploadHandoffRequest.kt
package app.mobiling.client.contract.attachment

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class AttachmentUploadHandoffRequest(
    val ownerType: String,
    val ownerId: String,
    val context: String? = null,
    val slot: String? = null,
    val isPrimary: Boolean = false,
    val title: String? = null,
    val description: String? = null,
    val altText: String? = null,
)
